Core/src/main/java/com/plotsquared/core/plot/flag/types/BlockTypeWrapper.java

```diff
@@ -86,15 +95,26 @@ public boolean accepts(final BlockType blockType) {
         }
     }
 
+    public BlockCategory getBlockCategory() {
+        if (this.blockCategory == null && this.blockCategoryId != null) { // only if name is available
+            this.blockCategory = Preconditions.checkNotNull(BlockCategory.REGISTRY.get(this.blockCategoryId));
```

Comment:
Will this cause a soft failure wherever getBlockCategory is called? It might be best to add an exception to the method so we're not destroying something we shouldn't in parent methods